libbeat/cmd/instance/beat.go

```diff
@@ -629,7 +629,7 @@ func (b *Beat) configure(settings Settings) error {
 	logp.Info("Beat ID: %v", b.Info.ID)
 
 	// initialize config manager
-	b.ConfigManager, err = management.Factory()(b.Config.Management, reload.Register, b.Beat.Info.ID)
```

Comment:
it's weird that this is missing

Reply:
`x-pack/libbeat/management/fleet` is missing entirely in branch

Reply:
check this PR and especially this test update to make libbeat pass
https://github.com/elastic/beats/pull/14835/files#diff-cf1cbef8b594350a8858a0682701458eR76

Reply:
weird le met check that. I have it locally I might not have pushed it but I still have issues with the libbeat system test.